tests/tuner/test_lr_finder.py

```diff
@@ -87,9 +88,11 @@ def test_trainer_reset_correctly(tmpdir):
         "max_steps",
     ]
     expected = {ca: getattr(trainer, ca) for ca in changed_attributes}
-    trainer.tuner.lr_find(model, num_training=5)
-    actual = {ca: getattr(trainer, ca) for ca in changed_attributes}
 
+    with no_warning_call(UserWarning, match="Please add the following callbacks"):
```

Comment:
The linked issue was closed saying that the warning disappeared. I would not make this change then

Reply:
fixed the description. I linked another issue by mistake.